pkg/pcl2pulumi/pcl2pulumi_test.go

```diff
@@ -52,6 +52,26 @@ func TestOperatorPy(t *testing.T) {
 	assertion.Equal(string(pyExpected), string(py), "python operator codegen is incorrect")
 }
 
+func TestMinReproPy(t *testing.T) {
+	assertion := assert.New(t)
+
+	pcl, err := ioutil.ReadFile("../../testdata/doubleQuotes.pp")
+	assertion.NoError(err)
+
+	_, err = Pcl2Pulumi(string(pcl), "../../testdata/k8sOperator/main", "python")
```

Comment:
I see that we're writing output in a few of these tests to `k8sOperator/main` which is kind of confusing. Can we use something that's more appropriate for the test? Maybe `testdate/doubleQuotes`? Same for `MiniReproTs` and `MiniReproCs`